apps/store/extensions/app/store-apis/apis/taxonomy.jag

```diff
@@ -140,45 +126,34 @@ require('/modules/store.js').exec(function (ctx) {
                         log.error('Error on retrieving taxonomy.Server is unable to prddocess the request.Please ' +
                                 'check your have entered a correct query', e);
                     }
-
                     return;
                 }
-
-
             } else if (uriMatcher.match(taxonomyPatternAssetType)) {
                 try {
                     if (jsonHeaderAccept >= 0 || acceptHeader == constants.ACCEPT_ALL) {
-
-
                         if (options['taxonomyId']) {
-                            taxaPathList = taxa.taxonomy.getTaxonomyName(options['taxonomyId'],
+                            taxaPathList = Taxonomy.taxonomy.getTaxonomyName(options['taxonomyId'],
```

Comment:
This should be Taxanomy.getTaxonomyName() not Taxonomy.taxonomy.getTaxonomyName. Please fix it.